src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 4 failures in `PowerShell-CI-linux`
Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got /home/vsts/work/1/a/bins/publish\libmi.so.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 64
64:         "${PSHOME}\${sourceDllName}" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got /tmp/tmpF5NQxh.tmp/testDllNativeFolder/linux-x64;/tmp/tmpF5NQxh.tmp/testDllNativeFolder/managed.dll.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 68
68:         (dir $root) -join ";" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got /tmp/tmpF5NQxh.tmp/testDllNativeFolder/linux-x64/nativedll.so.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 72
72:         (dir $archFolder) -join ";" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "Unable to load shared library '/tmp/tmpF5NQxh.tmp/linux-x64/nativedll.so' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: /tmp/tmpF5NQxh.tmp/linux-x64/nativedll.so: cannot open shared object file: No such file or directory"
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 80
80:         [TestNativeClass2]::LoadNative() | Should -Not -BeNullOrEmpty

```